packages/commonwealth/server/routes/updateChainCustomDomain.ts

```diff
@@ -0,0 +1,39 @@
+import { Response, NextFunction } from 'express';
+import { TypedRequestBody } from '../types';
+import { DB } from '../database';
+
+type updateCustomDomainReq = {
+  secret: string;
+  chain_id: string;
+  custom_domain: string;
+};
+
+const updateChainCustomDomain = async (
+  models: DB,
+  req: TypedRequestBody<updateCustomDomainReq>,
+  res: Response,
+  next: NextFunction
+) => {
+  if (!req.body) {
+    return next(new Error('invalid request body'));
+  }
+  // Verify Chain Exists
+  const { chain_id, custom_domain, secret } = req.body;
+  const chain = await models.Chain.findOne({
+    where: { id: chain_id },
+  });
+  if (!chain) return next(new Error('Chain not found.'));
```

Comment:
The error strings should be collected in an object at the top, like in most other routes.